template/apps/web/package.json

```diff
@@ -38,6 +40,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "7.34.2",
+    "react-markdown": "8.0.3",
```

Comment:
Do we really need it? 🤔 
I think it's quite specific and we can change design a little bit to avoid additional dependency